pkg/doc/verifiable/credential.go

```diff
@@ -369,26 +389,26 @@ type Credential struct {
 	Schemas        []TypedID
 	Evidence       *Evidence
 	TermsOfUse     []TypedID
-	RefreshService *TypedID
+	RefreshService []TypedID
 
 	CustomFields CustomFields
 }
 
 // rawCredential is a basic verifiable credential
 type rawCredential struct {
-	Context        interface{} `json:"@context,omitempty"`
-	ID             string      `json:"id,omitempty"`
-	Type           interface{} `json:"type,omitempty"`
-	Subject        Subject     `json:"credentialSubject,omitempty"`
-	Issued         *time.Time  `json:"issuanceDate,omitempty"`
-	Expired        *time.Time  `json:"expirationDate,omitempty"`
-	Proof          *Proof      `json:"proof,omitempty"`
-	Status         *TypedID    `json:"credentialStatus,omitempty"`
-	Issuer         interface{} `json:"issuer,omitempty"`
-	Schema         interface{} `json:"credentialSchema,omitempty"`
-	Evidence       *Evidence   `json:"evidence,omitempty"`
-	TermsOfUse     []TypedID   `json:"termsOfUse,omitempty"`
-	RefreshService *TypedID    `json:"refreshService,omitempty"`
+	Context        interface{}     `json:"@context,omitempty"`
+	ID             string          `json:"id,omitempty"`
+	Type           interface{}     `json:"type,omitempty"`
+	Subject        Subject         `json:"credentialSubject,omitempty"`
+	Issued         *time.Time      `json:"issuanceDate,omitempty"`
```

Comment:
Dima, from you user perspective, 	
issueDate := time.Date(time.Now().Year(), time.Now().Month(), time.Now().Day(),
		time.Now().Hour(), time.Now().Minute(), time.Now().Second(), 0, time.UTC) Validation only passed when i was entering this structure of time in the vc. Is that correct ?

Reply:
@talwinder50 Correct.